vmms/localDocker.py

```
@@ -282,7 +282,7 @@ def getPartialOutput(self, vm):
             config.Config.MAX_OUTPUT_FILE_SIZE,
         )
         output = subprocess.check_output(
```

Ruff (0.14.11)

`subprocess` call with `shell=True` identified, security issue

(S602)